common/djangoapps/microsite_configuration/tests/backends/test_base.py

```diff
@@ -130,3 +138,36 @@ def test_not_yet_implemented(self):
 
         with self.assertRaises(NotImplementedError):
             backend.get_all_orgs()
+
+
+@patch(
+    'microsite_configuration.microsite.BACKEND',
+    microsite.get_backend(
+        'microsite_configuration.backends.base.BaseMicrositeBackend', BaseMicrositeBackend
+    )
+)
+class BaseMicrositeBackendTests(TestCase):
+    """
+    Go through and test the BaseMicrositeBackend class for behavior which is not
+    overriden in subclasses
+    """
+    def test_enable_microsites_pre_startup(self):
+        """
+        Tests microsite.test_enable_microsites_pre_startup works as expected.
+        """
+        # remove microsite root directory paths first
+        settings.DEFAULT_TEMPLATE_ENGINE['DIRS'] = [
+            path for path in settings.DEFAULT_TEMPLATE_ENGINE['DIRS']
+            if path != settings.MICROSITE_ROOT_DIR
+        ]
+
+        with patch('microsite_configuration.backends.base.BaseMicrositeBackend.has_configuration_set',
```

Comment:
@felipemontoya I think it would be better if instead of patching `microsite_configuration.backends.base.BaseMicrositeBackend.has_configuration_set ` we patch `settings.MICROSITE_CONFIGURATION` and perform subsequent checks.